pkg/controller/stacks/install/stackinstall_test.go

```diff
@@ -49,6 +49,7 @@ const (
 	uid                     = types.UID(uidString)
 	resourceName            = "cool-stackinstall"
 	stackPackageImage       = "cool/stack-package:rad"
+	tsControllerImage       = "crossplane/ts-controller:0.0.0"
```

Comment:
Will this ever pull a real image? Would it make sense to use an image which is more obviously fake? At first I thought it was supposed to be a real image version.

Reply:
It would never pull a real image. I can change this to something more obviously fake.